frontends/election-manager/src/router_paths.ts

```diff
@@ -16,7 +16,8 @@ export const routerPaths = {
   definitionEditor: '/definition/editor',
   definitionContest: ({ contestId }: { contestId: ContestId }): string =>
     `/definition/contests/${contestId}`,
-  smartcards: '/cards',
+  smartcards: '/smartcards',
+  superAdminSmartcards: '/smartcards/super-admin-smartcards',
```

Comment:
I'm curious how you're thinking about routing. Another way to potentially do this would be to have the election cards/superadmin cards pages  be routed via a nested switch, since they are subpages of the smartcards page. Have you considered that option? Any pros/cons?

Reply:
My mental model for these pages so far has been that election card programming is the default state and super admin card programming is a special state (hence only the latter living at a sub-route). But I could definitely see a case for placing both at sub-routes (`/smartcards/election-cards` and `/smartcards/super-admin-smartcards`), though we'd have nothing unique to display at `/smartcards`.

I hesitate to use a nested `Switch` when the single top-level app `Switch` can do the trick, since I think a nested `Switch` is harder to reason about.

Regardless, let me play around with things a bit!

Reply:
Made some tweaks! https://github.com/votingworks/vxsuite/pull/2016/commits/5731ed15ca3a81d4e3d6711978f04dfa6b104362